pkg/systemd/logs.js

```diff
@@ -304,9 +326,10 @@ $(function() {
 
         var options = cockpit.location.options;
         if (options['service'])
-            match.push('_SYSTEMD_UNIT=' + options['service']);
+        match.push('_SYSTEMD_UNIT=' + options['service']);
         else if (options['tag'])
             match.push('SYSLOG_IDENTIFIER=' + options['tag']);
+        $('#journal-service').text(options['tag'] || _("All"));
```

Comment:
What about when there is no tag but there is a service? Is 'All' still right?

Reply:
Maybe I didn't understand your point but my answer to your question would be "yes". 
When there's no filter "all" services will be shown.